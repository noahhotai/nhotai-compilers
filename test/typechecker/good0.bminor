j: function integer (z: integer, y: char, t: float) = {
    foo: integer = z + 2 + 3;
    return foo;
}