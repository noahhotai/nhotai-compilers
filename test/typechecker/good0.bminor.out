Program parsed successfully.
z resolves to param 1
foo resolves to local 1
Program resolved successfully.
Program type checked successfully.
